### PR TITLE
Try out a lighter adblocking technique

### PR DESCRIPTION
**Pull request by sndrs**
this basically uses the same method as the current one, but only ads the element once, and on the main viewport canvas.

before, it was actually adding the element 5 times:

screen shot 2016-02-19 at 13 09 22

and `9999px` off screen.

this may have no effect, but i'm hopeful it may help iPad crashes

cc @jbreckmckye @regiskuckaertz

### CODE REVIEW

File: static/src/javascripts/projects/common/utils/detect.js [hunk 1]
```diff
@@ -405,38 +409,58 @@ define([
...
-    function adblockInUseSync() {
-        if (!detect.cachedAdblockInUse) {
-            var sacrificialAd = createSacrificialAd(),
-                contentBlocked = isHidden(sacrificialAd);
-            sacrificialAd.remove();
-            detect.cachedAdblockInUse = contentBlocked;
-            return contentBlocked;
-        }
+    var adblockInUseSync, getFirefoxAdblockPlusInstalledSync, createSacrificialAd;
+    if (userPrefs.get('lighter-adblock')) {
```
Comment (jbreckmckye) [inline static/src/javascripts/projects/common/utils/detect.js]: I actually probably wouldn't bother putting this in a separate branch - I'd just replace the original function entirely. This also make it easy to check the new approach remains accurate in the wild - we just have to look at Ophan and check adblockers are still being detected.

Comment (sndrs) [inline static/src/javascripts/projects/common/utils/detect.js]: ah ok yeah will do then

**Comment by sndrs** [comment]
updated to remove the pref

**Comment by regiskuckaertz** [comment]
Nice :mountain_cableway:

**Comment by sndrs** [comment]
is that a :+1:? :)

**Comment by jbreckmckye** [comment]
It's a :palm_tree: from me.

**Pull request merged**